src/styles/utilits/_variables.scss

```diff
@@ -0,0 +1,11 @@
+
+$card-width: 200px;
+$card-margin-right: 48px;
+$card-margin-bottom: 46px;
+$catalog-container-width: ($card-width * 4) + ($card-margin-right * 3);
```

Comment:
Remove magic numbers, add variables instead

Reply:
I did, but this is my variables, now I add more variables